FV3/compile

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -eu
```

Comment:
It might be helpful to add a comment describing what os being compiled (though is pretty obvious from reading the code, it's not obvious from the name of the file that this is compiling in several different modes)

Reply:
Agreed. Note, though, that this is simply a script that was provided with the original code and I simply resurrected it (must have been deleted by someone at some point).